ElementException: Element 'ReplicaModifications' does not have a match in class io.minio.messages.SourceSelectionCriteria

An exception occurred when I tried to call getBucketReplication .
ReplicationConfiguration config = minioClient.getBucketReplication( GetBucketReplicationArgs.builder() .bucket(bucketName).build() );
And stack trace:
Caused by: org.simpleframework.xml.core.ElementException: Element 'ReplicaModifications' does not have a match in class io.minio.messages.SourceSelectionCriteria at line 1
	at io.minio.Xml.unmarshal(Xml.java:55)
	at io.minio.MinioAsyncClient.lambda$getBucketReplication$56(MinioAsyncClient.java:2493)
	
thank you  for your answer！

--- a/api/src/main/java/io/minio/messages/ReplicaModifications.java
+++ b/api/src/main/java/io/minio/messages/ReplicaModifications.java
@@ -1,0 +1,37 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2022 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio.messages;
+
+import java.util.Objects;
+import javax.annotation.Nonnull;
+import org.simpleframework.xml.Element;
+import org.simpleframework.xml.Root;
+
+/** Helper class to denote replica modifications information for {@link SourceSelectionCriteria}. */
+@Root(name = "ReplicaModifications")
+public class ReplicaModifications {
+  @Element(name = "Status")
+  private Status status;
+
+  public ReplicaModifications(@Nonnull @Element(name = "Status") Status status) {
+    this.status = Objects.requireNonNull(status, "Status must not be null");
+  }
+
+  public Status status() {
+    return this.status;
+  }
+}

--- a/api/src/main/java/io/minio/messages/SourceSelectionCriteria.java
+++ b/api/src/main/java/io/minio/messages/SourceSelectionCriteria.java
@@ -23,13 +23,27 @@ import org.simpleframework.xml.Root;
 /** Helper class to denote source selection criteria information for {@link ReplicationRule}. */
 @Root(name = "SourceSelectionCriteria")
 public class SourceSelectionCriteria {
+  @Element(name = "ReplicaModifications", required = false)
+  private ReplicaModifications replicaModifications;
+
   @Element(name = "SseKmsEncryptedObjects", required = false)
   private SseKmsEncryptedObjects sseKmsEncryptedObjects;
 
   public SourceSelectionCriteria(
       @Nullable @Element(name = "SseKmsEncryptedObjects", required = false)
-          SseKmsEncryptedObjects sseKmsEncryptedObjects) {
+          SseKmsEncryptedObjects sseKmsEncryptedObjects,
+      @Nullable @Element(name = "ReplicaModifications", required = false)
+          ReplicaModifications replicaModifications) {
     this.sseKmsEncryptedObjects = sseKmsEncryptedObjects;
+    this.replicaModifications = replicaModifications;
+  }
+
+  public SourceSelectionCriteria(@Nullable SseKmsEncryptedObjects sseKmsEncryptedObjects) {
+    this(sseKmsEncryptedObjects, null);
+  }
+
+  public ReplicaModifications replicaModifications() {
+    return this.replicaModifications;
   }
 
   public SseKmsEncryptedObjects sseKmsEncryptedObjects() {

